### ISSUE TITLE
Major: Convolutional Neural Network Integration

### ISSUE THREAD

**Issue by FramedStone** [FramedStone/SassyNic#39]
### Purpose
The purpose of this `feature` is to utilise `Convolutional Neural Network (CNN)` to compare current term's timetable with all timetable combinations generated by SassyNic - Student, and sort the display list accordingly based on `Similarity`

### Brief Structure
- `Training Dataset` = timetable combinations generated by SassyNic - Student
- Adjustable `Epochs`
- Adjustable `Learning Rate`
- Choosable `Kernel` between `2x2` and `3x3`
- `Button` to choose whether to use this feature
